xAxis : [0,-1,0]
yAxis : [0,0,1]
zAxis : [-1,0,0]
centerPoint : [-1.0,0.0,0.8]

